.github/workflows/main.yml

```diff
@@ -30,6 +30,32 @@ jobs:
   Test_sqlite:
     uses: formancehq/gh-workflows/.github/workflows/golang-test.yml@main
 
+  bench:
```

Comment:
Bench are very slow, I don't think we have to add them to the CI.
@flemzord ?

Reply:
I added the benchmarks in a separate workflow_dispatch, to run them only manually. What do you think?